### DISCUSSION TITLE
Library panels/dashboard across organizations

### DISCUSSION THREAD

**Discussion by wkneewalden** [grafana/grafana#37065] [Dashboards, Panels & Visualizations]
Can the functionality of library panels be extended across organizations? Can the library functionality be extended to entire dashboards? Being able to have only one set of dashboard/panels in one organization that needs to be updated would save a lot of time on automation.

**Comment by pkolyvas** [comment]
@wkneewalden In the case you describe, do you use the exact same dashboards, panels, queries across many organizations? Can you expand a little bit to help us understand the use case/workflow further?

**Reply by wkneewalden** [reply]
It would be same dashboards used across organizations. Each organization would have a couple of dashboards that use same template, with same metrics queries containing variables. There would be a few core variables used to distinguish organizations, but all other variables used in panel queries would trace back to these core variables, preserving the similarity of dashboards between organizations. 

If library panels could trace their effects across organizations, it would make hotfix changes to each organization panels easier via the console in one organization than say replacing dashboards in each org using an API script which is current technique.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
